syserol/dataImport.py

```diff
@@ -85,18 +85,35 @@ def createCube():
 
     IGG = importIGG()
     glycan, dfGlycan = importGlycan()
-    glyCube = np.full([len(subjects), len(glycan)], np.nan)
+    func = importFunction()
```

Comment:
Why not inner join this with `dfGlycan`, so you only have one table to worry about?